en/mozorg/about/governance/policies/reporting.ftl

```diff
@@ -16,13 +16,14 @@ reporting-how-to-give-a-report = How to Give a Report
 reporting-if-you-believe-someone = If you believe someone is in physical danger, call your local emergency number.
 reporting-if-you-have-a-report-by = If you have a report <strong>by <em>and</em> about</strong> a contributor (for example, the report is made <strong>by</strong> one contributor <strong>about</strong> another contributor), then you should make your report at the <a href="{ $community_hotline }">Community Participation Guidelines hotline</a>.
 reporting-if-you-have-a-report-by-email = If you have a report <strong>by <em>and</em> about</strong> a contributor (for example, the report is made <strong>by</strong> one contributor <strong>about</strong> another contributor), then you should make your report at the <a href="{ $mailto_report }">cpg-report@mozilla.com</a>.
-reporting-if-you-have-a-report-involving = If you have a report <strong>involving an employee, contractor, or vendor</strong> (for example, the report is made <strong>by</strong> an employee or is <strong>about</strong> an employee), then you should report at the <a href="{ $employee_hotline }">{ -brand-name-mozilla } Employee hotline</a>.
+reporting-if-you-have-a-report-involving-v2 = If you have a report by a contributor about <strong>an employee</strong>, then you should report at the <a href="{ $employee_hotline }">{ -brand-name-mozilla } Employee hotline</a>. If you are an employee and need to submit a report, please consult { -brand-name-mozilla }'s internal policies and procedures to understand the reporting options available to you.
```

Comment:
Do we need to keep the old string and put an expiration date on it like how we manage string updates?